helm/app-operator-chart/templates/network-policy.yaml

```diff
@@ -0,0 +1,20 @@
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  namespace: {{ .Values.namespace }}
+  name: {{ .Values.name }}-network-policy
```

Comment:
Isn't it good to add `app` label also here?

Reply:
Even more, like this https://github.com/giantswarm/net-exporter/blob/master/helm/net-exporter/templates/np.yaml#L7-L8